djangocms_versioning_filer/fields/file.py

```diff
@@ -112,6 +114,18 @@ def widget_attrs(self, widget):
         widget.required = self.required
         return {}
 
+    def clean(self, value):
+        value = super().clean(value)
+
+        if value is not None and self.required is False:
+            published_exists = Version.objects.filter_by_grouper(value).filter(
+                state="published"
+            ).exists()
+            if not published_exists:
+                raise ValidationError("No published version currently exists, publish it first")
+
+        return value
+
     def to_python(self, value):
         obj = super().to_python(value)
```

Comment:
before this statement, we could filter the queryset so that making sure that we get the correct file. So in essence we wouldn't do super at all.